File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        frq = [0]*26
        for ch in s:
            frq[ord(ch)-ord('a')]+=1

        for ch in t:
            frq[ord(ch)-ord('a')]-=1

        for f in frq:
            if(f!=0):
                return False

        return True

        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] frqArray = new int[26];
        for(char ch:s.toCharArray()){
            frqArray[ch-'a']++;
        }
        for(char ch:t.toCharArray()){
            if(frqArray[ch-'a']==0){
                return false;
            }
            else{
                frqArray[ch-'a']--;
            }
        }
        for(char ch:s.toCharArray()){
            if(frqArray[ch-'a']!=0){
                return false;
            }
        }
        return true;
    }
}